DIFFERENCES.md

```diff
@@ -0,0 +1,82 @@
+# Important Differences
+The [JCA/JCE](https://docs.oracle.com/en/java/javase/11/security/java-cryptography-architecture-jca-reference-guide.html) specification does not completely define all behaviors by a given provider.
+Thus, though the Amazon Corretto Crypto Provider (ACCP) is fully compliant with the JCE, it does differ in behavior from the default Java providers in several ways.
+The following list is not exhaustive but is intended to capture the most important differences to our consumers.
```

Comment:
```suggestion
The following list of behavioral differences is not exhaustive, but is intended to capture the most important differences.
```

I think you meant "customers," not "consumers," but either way, I'd keep it in the second person.